components/BottomNavigation/examples/BottomNavigationNilBadges.swift

```diff
@@ -46,6 +46,7 @@ class BottomNavigationNilBadges : UIViewController {
 
     // Add items to the bottom navigation bar.
     let tabBarItem1 = UITabBarItem(title: "Home", image: UIImage(named: "Home"), tag: 0)
+    tabBarItem1.accessibilityValue = "New items"
```

Comment:
Should this be "Home"?

Reply:
It goes "\(tabBarItem.title) \(accessibilityValue) tab" so that would be "Home home tab" for this we need to indicate that there is a badge.

Reply:
Does it get removed when the user taps on it?

Reply:
What is "it" in this context? Neither the badge or the label get removed.

Reply:
Why is "Home New items tab" better in this case? I'm not sure I understand the problem this PR is solving.

Reply:
The problem we are trying to solve is, indicate to VoiceOver users that there is a badge there. My assumption is that a badge typically means that their are "new items".

Reply:
I see. This feels like it may warrant a bit more depth in analysis on what the correct approach here is. I'd expect the value of the badge to be what's read out here in some manner.

Reply:
Removing myself from LGTM because I'm not confident in this solution.

Reply:
I can add a value then it would say "Home 5 items tab" and the badge value would be 5.

Reply:
The example is showcasing the "dot" badge. I was asking if the dot disappears once the item is selected?

Reply:
It does not, do you feel that it should?

Reply:
Spoke offline with @romoore and updated the PR to clear the badge and accessibility value when the tab is selected.